modules/root/root_api.rb

```diff
@@ -12,6 +12,27 @@ class Proxy::RootApi < Sinatra::Base
     end
   end
 
+  get "/features/v2" do
+    begin
+      enabled_plugins = ::Proxy::Plugins.instance.select do |p|
+        p[:name] != :foreman_proxy && \
+          p[:state] == :running && \
+          p[:class].ancestors.include?(::Proxy::Plugin)
+      end
+      content_type :json
+      enabled_plugins.sort { |x, y| x[:name] <=> y[:name] }.map do |p|
+        {
+          'name' => p[:name],
+          'capabilities' => p[:capabilities],
+          'http_enabled' => p[:http_enabled],
+          'https_enabled' => p[:https_enabled],
+        }
+      end.to_json
+    rescue => e
+      log_halt 400, e
+    end
+  end
+
```

Comment:
Oh there you have it. I vote for lowercasing them - they are symbols anyway, they read better lowercase.

Reply:
I see them a bit like constants and constants are typically uppercase. I do think that we should upper case them in the REST API.